guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,146 @@
+| millibit     | mBTC         | 0.00 100 000      |
+| bit          | μBTC         | 0.00 000 100      |
+| satoshi      | sat          | 0.000 000 01      |
+| millisatoshi | msat         | 0.000 000 000 01  |
+
+For more information, see the Bitcoin Wiki:
+
+- [Units](https://en.bitcoin.it/wiki/Units)
+- [Satoshi](https://en.bitcoin.it/wiki/Satoshi_(unit))
+
+#### Recommended interaction
+
+When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satosho for Lightning wallets. Due to the challenging nature of scanning a monetary value with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings section) as well as contextually, whenever the value is primarily displayed.
+
+Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while the opposite can be true for savings-focused applications.
+
+###### Setting the preferred unit
+
+Applications should try to establish smart defaults that work for the majority of users, while offering convenient ways to customize settings.
+
+{% include image-gallery.html pages = page.images_app %}
+
+###### Changing the unit display contextually
+
+User should always be able to change unit contextually. Additionally, for the value to have meaning to most users today, the option to select the local currency should also be readily available.
+
+{% include image-gallery.html pages = page.images_home %}
+
+#### Formatting units
+
+There are many different ways of formatting numbers and currency units across the world. Bitcoin applications should be sensitive to these standards by adapting the formatting to the user's locale. A simple example are digit group and decimal separators. Based on where you live you may be accustomed to one of these formatting standards, but be less comfortable with the others:
+- `10 000,00`
+- `10.000,00`
+- `10,000.00`
+- `10'000,00`
+- `10,000·00`
+
+For more examples and information, see Wikipedia:
+
+- [Currency symbol](https://en.wikipedia.org/wiki/Currency_symbol)
+- [Unit prefix](https://en.wikipedia.org/wiki/Unit_prefix)
+- [Decimal separator](https://en.wikipedia.org/wiki/Decimal_separator)
+
+#### Visual styling
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/visual-formatting-examples.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/visual-formatting-examples@2x.png"
+   alt-text = "Bitcoin island travel map"
+   width = 400
+   height = 195
+   layout = "float-right-desktop"
+%}
+
+Type choices, spacing, color, and other details also affect the ease at which users can understand written amounts. Take a look at the two examples to see which ones are easier to read.
```

Comment:
@GBKS you have some good settings you told me about when setting numbers. it could be worth mentioning a few of them here by name so people can know what to lookup.

Reply:
I could bake those into the graphic next to this copy, the same way I did in the header graphic.